channels-rke2.yaml

```diff
@@ -234,6 +234,15 @@ releases:
       harvester-csi-driver:
         repo: rancher-rke2-charts
         version: 0.1.400
+  - version: v1.21.8+rke2r1
+    minChannelServerVersion: v2.6.1-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs:
+      <<: *serverArgs-v1-21-7-rke2r2
+    agentArgs:
+      <<: *agentArgs-v1-21-7-rke2r2
+    charts:
+      <<: *charts-v1-21-7-rke2r2
```

Comment:
I think we just need top bump harvester versions here (and in the 1.22 part) according to: 
https://github.com/rancher/rke2/blob/v1.21.8%2Brke2r1/Dockerfile#L112-L113

Reply:
right, forgot we added that at the last minute